src/ddsrt/src/rusage/posix/rusage.c

```diff
@@ -109,7 +109,7 @@ ddsrt_getrusage_anythread (
     return DDS_RETCODE_ERROR;
   if ((fp = fopen (file, "r")) == NULL)
     return DDS_RETCODE_NOT_FOUND;
-  enum { ERROR, READ_HEADING, SKIP_TO_EOL, READ_VCSW, READ_IVCSW } state = READ_HEADING;
+  enum { ERROR = 1, READ_HEADING, SKIP_TO_EOL, READ_VCSW, READ_IVCSW } state = READ_HEADING;
```

Comment:
Why does ``ERROR`` have to be 1 instead of 0?

Reply:
In the path of this error in coverity states that the value of "state" must be at least 1. ERROR is the lowest value of state.

Reply:
I get it. Coverity is correct in the sense that in ``switch(state)``, ``state`` ≠ ``ERROR``, so ≥1 given the way enumerated types in C work. The ``case ERROR`` is simply there to make it clear to the compiler (which typically doesn't perform the same level of analysis that Coverity does) that all possible cases have been considered.

One could eliminate the ``ERROR`` state entirely by doing ``{fclose(fp);return DDS_RETCODE_ERROR;}`` where it enters that state. But I like the structure with an ERROR state better here …

Just tell Coverity that this is intentional instead of changing the code.